onnxscript/rewriter/onnxruntime/transformers/multihead_attention.py

```diff
@@ -592,3 +623,89 @@ def phi_attention(
             phi_attention
         ).to_function_proto()
         return ir.serde.deserialize_function(function_proto)
+
+
+class MHAStableDiffusionUnetRewriteRule(AttentionRewriteRule):
+    FUNCTION_KEYWORD = "Attention"
+    PACKAGE_NAME = "diffusers"
+    _version_controller = function_rule.VersionController()
+
+    def __init__(self) -> None:
+        super().__init__()
+
+    @_version_controller.register_version()
+    def _fusion(self, function: ir.Function) -> ir.Function:
+        if len(function.inputs) != 6 and len(function.inputs) != 7:
```

Comment:
Explain where this 6/7 inputs is defined? Perhaps a link to transformers code etc?

Reply:
I added the expected input names.